src/transformers/pipelines/visual_question_answering.py

```diff
@@ -118,20 +136,55 @@ def preprocess(self, inputs, padding=False, truncation=False):
         model_inputs.update(image_features)
         return model_inputs
 
-    def _forward(self, model_inputs):
-        model_outputs = self.model(**model_inputs)
+    def _forward(self, model_inputs, top_k=5, max_new_tokens=None, num_beams=None):
```

Comment:
```suggestion
    def _forward(self, model_inputs, **generate_kwargs):
```
I think the forward should just accept a `generate_kwargs` like the [text generation pipeline](https://github.com/huggingface/transformers/blob/3658488ff77ff8d45101293e749263acf437f4d5/src/transformers/pipelines/text_generation.py#LL231C38-L231C57)